### ISSUE TITLE
Object "names" doesn't support spaces in its keys

### ISSUE THREAD

**Issue by rindeal** [Mikulas/csfd-api#14]
Eg. http://csfdapi.cz/movie/234460?type=html 

``` json
 "names": {
    "cs": "Skyfall",
    "velká": "Skyfall",
    "sk": "Skyfall"
  }
```

`velká` should be `velk[aá] brit[aá]nie`, `velká_brit[aá]nie` or `vb`

**Comment by Mikulas** [comment]
Thanks, this is properly fixed on master (WIP 2.0) https://github.com/Mikulas/csfd-api/blob/master/src/Csfd/Parsers/Movie.php#L178
I hope to upgrade web api to 2.0 as soon as all features from the 1.0 branch are implemented.

**Comment by rindeal** [comment]
Ahaa, i see now, ok, in the meantime, I'll try to setup a local instance, thanks.
